src/logAppender.js

```diff
@@ -0,0 +1,38 @@
+import {
+	AppInsights
+} from "applicationinsights-js";
+
+export default class AppInsightsAppender {
+	constructor(plugin) {
+		this._plugin = plugin;
+	}
+
+	debug(logger, message) {
+		if (!this._plugin.initialized) {
```

Comment:
Line 136 in `application-insights` already checks if the plugin has been initialized. I don't think the checks in `logAppender` are necessary and `logAppender` probably shouldn't care about the plugin anyway, right?